CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ### Enhancements
 
+* Automatic ANR detection is now enabled by default  
```

Comment:
Since this is a platform-specific thing, I try to over-explain / link to related docs for stuff like this:

```suggestion
* [Android] Automatic [App Not Responding](https://developer.android.com/topic/performance/vitals/anr) (ANR) detection is now enabled by default  
```